cypress/e2e/SP_21_Camera_Based_Event_Creation.cy.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This tests confirms that the client-side file selection features are working.
+ * It does not confirm that the server is function.
+ */
+
+describe('Test Camera Based Event Creation [SP-21]', function() {
+
+    beforeEach(() => {
+        cy.visit(Cypress.env('baseUrl'));
+        cy.login("testEmail_1","testPassword_1");
+    })
+
+    it('Add Course using Upload Picture Button', function() {
+        cy.get('[data-test="uplaodPhotoEventButtonCalenderPage"]').click();
+        cy.get('#fileSelectButton').invoke('show').selectFile('cypress/photos/Soen_385.png');
```

Comment:
This test is incomplete. It uploads an image, but does not test that the event data from the image is populated in the subsequent addEvent drawer.

Reply:
True! I couldn't get it working with my image, can we do a walkthrough together with another image?

Reply:
Use the following image to populate all the fields that are supported for this user story:
![image](https://user-images.githubusercontent.com/72574688/219255404-b5fc345d-3d60-4a94-9465-8c4a5afa881f.png)

Follow the same steps, but you will noticed that after pressing 'Upload', a new drawer will be presented with the information shown in the image.